src/fundamental/chid-fundamental.c

```diff
@@ -147,6 +147,21 @@ const uint32_t chid_smbios_table[CHID_TYPES_MAX] = {
                (UINT32_C(1) << CHID_SMBIOS_BASEBOARD_PRODUCT),
 
         [14] = (UINT32_C(1) << CHID_SMBIOS_MANUFACTURER),
+
+        /* Extra CHIDs */
```

Comment:
hmpf. we we are somewhat stepping in the territory of msft's spec here. I guess it's OK to do this as long as we never claim this was CHID type 15 and so on, hence I#d prefer if we'd make the base of 15 a bit more opaque. for example, rewrite these entries here as:

```c
[EXTRA_CHID_BASE+0] = (UINT32_C(1) …
[EXTRA_CHID_BASE+1] = …
```

you get the idea.


and then somewhere define `EXTRA_CHID_BASE` as 15. And add a very clear comment that these are non-standard chids, and subject to change and renumeration at any time.